project/source/Scripts/LabLogSingleton.gd

```diff
@@ -11,43 +11,43 @@ extends Node
 #each log is stored as dictionaries like this:
 #{'message': "something", 'hidden': false, 'popup': false}
 #log data will be used by other things. This node just stores them and notifies others when they change.
-var logs = {}
-signal NewMessage(category, newLog)
+var logs: Dictionary = {}
+signal NewMessage(category: String, newLog: LabLog)
```

Comment:
I think `LabLog` is a singleton object rather than a type. Based on this signal is used, I think it's likely that `newLog` is a `Dictionary`.

Reply:
I realize that this change was effectively cherry picked from `dev`, so this can be resolved elsewhere.